### ISSUE TITLE
types missing in 2.1.0 release?

### ISSUE THREAD

**Issue by cexbrayat** [asciidoctor/asciidoctor.js#828]
Thank you for the 2.1.0 release!

I was happy to see that it now provides a TypeScript Declaration File but the TS compiler is not finding it, and I can't see it in the installed module:

```
npm ls @asciidoctor/core                                                               
exercises@1.0.0
└── @asciidoctor/core@2.1.0

tree node_modules/@asciidoctor/core                                                      
node_modules/@asciidoctor/core
├── LICENSE
├── README.md
├── dist
│   ├── browser
│   │   ├── asciidoctor.js
│   │   └── asciidoctor.min.js
│   ├── css
│   │   └── asciidoctor.css
│   ├── graalvm
│   │   └── asciidoctor.js
│   └── node
│       └── asciidoctor.js
└── package.json

5 directories, 8 files
```

Am I missing something obvious?

**Comment by cexbrayat** [comment]
Ha I now see that opened #827 sorry...
I'll let this open to track the issue if someone else stumbles on it, ut feel free to close it if you prefer.

**Comment by Mogztter** [comment]
Thanks for this quick feedback :wink: 
No it's my bad, I didn't include the file in the npm package :sweat: 

I will release 2.1.1 shortly. I'm just waiting a bit to make sure that there's not another big issue to fix in 2.1.1.
